test/getCommands.spec.js

```diff
@@ -7,13 +7,15 @@ const mockFs = require('mock-fs');
 const sinon = require('sinon');
 
 const commands = require('./fixtures/commands');
-const testPluginPath = path.join(process.cwd(), 'node_modules', 'rnpm-plugin-test');
-const testPluginPath2 = path.join(process.cwd(), 'node_modules', 'rnpm-plugin-test-2');
+const nestedPluginPath = path.join(process.cwd(), 'node_modules', 'rnpm-plugin-test');
+const nestedPluginPath2 = path.join(process.cwd(), 'node_modules', 'rnpm-plugin-test-2');
+const flatPluginPath = path.join(process.cwd(), '..', 'rnpm-plugin-test');
+const flatPluginPath2 = path.join(process.cwd(), '..', 'rnpm-plugin-test-2');
 const pjsonPath = path.join(__dirname, '..', 'package.json');
```

Comment:
We will have to mock `__dirname` in test suites I think to point to `process.cwd(), 'node_modules', 'rnpm'` in suites where we test `it should work as local dependency`.